mods/ra/maps/chernobyl/rules.yaml

```diff
@@ -11,26 +11,4 @@ World:
 		Tileset: TEMPERAT
 		Filename: plains.pal
 		ShadowIndex: 3, 4
-	PaletteFromFile@player:
-		Name: player
-		Filename: temperat.pal
-		ShadowIndex: 4
-	PaletteFromFile@chrome:
-		Name: chrome
-		Filename: temperat.pal
-		ShadowIndex: 3
-		AllowModifiers: false
-	PaletteFromFile@cursor:
-		Name: cursor
-		Filename: temperat.pal
-		AllowModifiers: false
-		CursorPalette: true
-	PaletteFromFile@effect:
-		Name: effect
-		Filename: temperat.pal
-		ShadowIndex: 4
-	PaletteFromFile@colorpicker:
-		Name: colorpicker
-		Filename: temperat.pal
-		ShadowIndex: 4
-		AllowModifiers: false
+
```

Comment:
This file has two new lines at the end now. (We only want one.)

Reply:
Removed it.